python/GangaDirac/Dockerfile

```diff
@@ -1,34 +1,28 @@
-FROM alexanderrichards/dirac_ui:latest
+# FROM taking build arg requires at least
+#   Docker version 17.05.0-ce, build 89658be
+ARG ganga_version=latest
+FROM ganga-core:$ganga_version
```

Comment:
have now removed reference to 6.5.2 instead defaulting to the latest. This however should be a build arg such that the default is never used for our purposes.